.github/workflows/validate.yml

```diff
@@ -0,0 +1,43 @@
+name: Validate code
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  setup:
+    uses: equinor/dm-models/.github/workflows/_setup.yml@main
+  lint:
+    name: Lint
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get base files
+        id: get-base-files
+        uses: actions/cache@v3
```

Comment:
Not too familiar with this package, but I assume it does its job. Using [setup-node](https://github.com/actions/setup-node) in my projects. You might want to check it out too.

Reply:
not going to lie, i copy/pasted a lot from the data marketplace workflow 🙈

Reply:
Hehe, copying and pasting is a good thing as it can save us time.